ansible/templates/centos7/deps_ice3.6.sh.j2

```diff
@@ -1,5 +1,5 @@
 {% if doc %}
-Install ZeroC IcePy {{ ice_version }}. IcePy is managed by `PyPI <https://pypi.python.org/pypi>`_, a package management system used to install and manage software packages written in Python. IcePy will be installed as part of the OMERO.web requirements::
+Install ZeroC IcePy {{ ice_version }}. IcePy is managed by :pypi:`PyPI <>`, a package management system used to install and manage software packages written in Python. IcePy will be installed as part of the OMERO.web requirements::
```

Comment:
this is probably a PyPI change which we cannot do much for, but when the link https://pypi.python.org/pypi/ is clicked, the experience there is not optimal (It says: Index of Packages[Deprecated]). Maybe we could link to some more helpful page ?

Reply:
We might not be able to use :pypi: I will check: https://pypi.python.org/pypi no ``/`` is fine

Reply:
Yes, but the inbuilt link in the doc page has the slash at the end. Maybe this needs changing.

Reply:
I will look where it is used in the doc and adjust accordingly (outside the scope of this PR since it will be adjusted in ome-doc)
Thanks for checking

Reply:
Problem fixed in the doc. I will use the new external link in this PR
https://github.com/openmicroscopy/ome-documentation/pull/1761